tests/integration/cli/anomaly/test_anomaly_classification.py

```diff
@@ -44,6 +34,38 @@
 templates_ids = [template.model_template_id for template in templates]
 
 
+class TestAnomalyClassificationModelTemplates:
+    @e2e_pytest_component
+    def test_model_category(self):
+        stat = {
+            ModelCategory.SPEED: 0,
+            ModelCategory.BALANCE: 0,
+            ModelCategory.ACCURACY: 0,
+            ModelCategory.OTHER: 0,
+        }
+        for template in templates:
+            stat[template.model_category] += 1
+        assert stat[ModelCategory.SPEED] == 1
+        assert stat[ModelCategory.BALANCE] <= 1
+        assert stat[ModelCategory.ACCURACY] == 1
+
+    @e2e_pytest_component
+    def test_model_status(self):
```

Comment:
It seems like the code of the last two tests is duplicated across all the tasks. Can we extract it to an abstract class?

Reply:
Thank you for the suggestion. Done :)